selfdrive/car/subaru/carstate.py

```diff
@@ -53,9 +53,13 @@ def update(self, cp, cp_cam, cp_body):
     steer_threshold = 75 if self.CP.carFingerprint in PREGLOBAL_CARS else 80
     ret.steeringPressed = abs(ret.steeringTorque) > steer_threshold
 
-    cp_cruise = cp_body if self.car_fingerprint in GLOBAL_GEN2 else cp
-    ret.cruiseState.enabled = cp_cruise.vl["CruiseControl"]["Cruise_Activated"] != 0
-    ret.cruiseState.available = cp_cruise.vl["CruiseControl"]["Cruise_On"] != 0
+    if self.car_fingerprint == CAR.FORESTER_2022:
+      ret.cruiseState.enabled = cp_cam.vl["ES_DashStatus"]['Cruise_Activated'] != 0
+      ret.cruiseState.available = cp_cam.vl["ES_DashStatus"]['Cruise_On'] != 0
```

Comment:
did we verify that these signals are good representations of the cruise state? See my concerns in this PR: https://github.com/commaai/openpilot/pull/25378

Reply:
We are looking into disable disengage on gas press not working issue that is probably related to second case in https://github.com/commaai/openpilot/pull/25378

Reply:
We switched to using ES_Status Cruise_Activated which behaves identically to CruiseControl Cruise_Activated signal on older models. That also fixed no disengage on gas press